picard_mark_duplicates_2.8.1/picard_mark_duplicates_2.8.1.cwl

```diff
@@ -0,0 +1,166 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  dct: 'http://purl.org/dc/terms/'
+  doap: 'http://usefulinc.com/ns/doap#'
+  foaf: 'http://xmlns.com/foaf/0.1/'
+id: picard_mark_duplicates_2.8.1
+baseCommand:
+  - java
+inputs:
+  - id: memory_per_job
+    type: int?
+    doc: Memory per job in megabytes
+  - id: memory_overhead
+    type: int?
+    doc: Memory overhead per job in megabytes
+  - id: number_of_threads
+    type: int?
+  - id: input
+    type: File
+    inputBinding:
+      position: 0
+      prefix: I=
+      separate: false
+    doc: Input file (bam or sam).  Required.
+  - id: output
+    type: string
+    inputBinding:
+      position: 0
+      prefix: O=
+      separate: false
+      valueFrom: '$(inputs.input.basename.replace(/.bam/, ''_md.bam''))'
```

Comment:
I'm a bit worried that these double quotes aren't supposed to be there, how does it show up in the expression editor in rabix?

Reply:
Works with cwltool and Rabix.